Extend/Model/Order.php

```diff
@@ -45,12 +45,31 @@ public function __construct()
      * Checks if the Paymenttype is one of the module's
      *
      * @return bool
-     *
-     * @SuppressWarnings(PHPMD.Coverage)
      */
     public function isModulePaymentType()
     {
-        return in_array($this->getPaymentType()->oxuserpayments__oxpaymentsid->value, $this->_aModulePaymentTypes);
+        return in_array($this->oxorder__oxpaymenttype->value, $this->_aModulePaymentTypes);
+    }
+
+    /**
+     * Checks if the payment is pending
+     *
+     * @return bool
+     */
+    public function isPaymentPending(): bool
```

Comment:
Why calling the methods payment? I would rather use isModuleOrderPending or isModuleStatusPending. Also for isPaymentSuccessful.

Reply:
It should show the payment state, not the order state